Makefile

```diff
@@ -7,19 +7,22 @@ rebar_cmd = $(rebar) $(profile:%=as %)
 GEN_INPUT = include/*.hrl priv/kpro_gen.escript priv/kafka.bnf priv/kpro_scanner.xrl priv/kpro_parser.yrl
 GEN_CODE = src/kpro_schema.erl
 
-.PHONY: gen-code gen-clean kafka-bnf
-
+# results of this target are to be commited in this repo
+.PHONY: kafka-bnf
 kafka-bnf:
 	@cd priv/kafka_protocol_bnf && gradle run
+	@cat priv/kafka.bnf | grep "#.*ApiKey" | awk '{print "{"$$2" "$$3"}."}' | sed -r 's/([A-Z])/_\L\1/g' | sed 's/{_/{/' | sort -hk2 > priv/api-keys.eterm
```

Comment:
Those double dollar signs look fragile (a bit of "how many quoting levels do I need?").  I assume it works, just makes me nervous.